acq4/devices/MicroManagerCamera/mmcamera.py

```diff
@@ -426,7 +439,9 @@ def getParam(self, param):
             'bitDepth': 'PixelType',
         }.get(param, param)
         with self.camLock:
-            val = self.mmc.getProperty(self.camName, str(paramTrans))
+            if paramTrans == 'Exposure':
+                val = self.mmc.getExposure(self.camName) # workaround for CellCam
+                val = self.mmc.getProperty(self.camName, str(paramTrans))
```

Comment:
This line probably needs to be in an else block?

Reply:
You're right, my bad!
```suggestion
            else:
                val = self.mmc.getProperty(self.camName, str(paramTrans))
```